site/globalDetailsOnDemand.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react"
+import { tippy } from "@tippyjs/react"
+import { BAKED_BASE_URL } from "../settings/clientSettings.js"
+import { renderToStaticMarkup } from "react-dom/server.js"
+import { EnrichedBlockText } from "@ourworldindata/utils/dist/owidTypes.js"
+import { ArticleBlocks } from "./gdocs/ArticleBlocks.js"
+
+export async function runGlobalDetailsOnDemand() {
+    const details = await fetch(`${BAKED_BASE_URL}/dods.json`, {
+        method: "GET",
+        credentials: "same-origin",
+        headers: {
+            Accept: "application/json",
+        },
+    }).then((res) => res.json())
+
+    ;(window as any).details = details
+
+    document.addEventListener("mouseover", handleEvent)
+    document.addEventListener("touchstart", handleEvent)
```

Comment:
could add `{passive: true}` to these listeners to improve performance, see https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener